conf/kairosdb.properties

```diff
@@ -92,4 +92,11 @@ kairosdb.datastore.cassandra.cache.tag_value.size=1024
 # has filled over a 90-day window:
 kairosdb.datastore.cassandra.use_new_split_index_read=true
 kairosdb.datastore.cassandra.use_new_split_index_write=true
-kairosdb.datastore.cassandra.new_split_index_start_time_ms=9223372036854775807
\ No newline at end of file
+kairosdb.datastore.cassandra.new_split_index_start_time_ms=9223372036854775807
+
+#Timeout for Cassandra datapoints queries
+kairosdb.datastore.datapoints.read.timeout=20000
+#Threadpool configuration for kairosdb read
+kairosdb.datastore.datapoints.read.threadpool.max.size=100
+kairosdb.datastore.datapoints.read.threadpool.core.size=10
+kairosdb.datastore.datapoints.read.thread.keep.alive.time.milliseconds=500
```

Comment:
Naming nitpick: .s separate categories, not name components, so it should be something like:

```properties
kairosdb.datastore.datapoints.read.thread_keep_alive_time_milliseconds
```

Or if you want to keep threadpool as a category, then something like:

```bash
kairosdb.datastore.datapoints.read.threadpool.keep_alive_time_milliseconds
```

The existing imports will work all the same.